yarn.lock

```diff
@@ -1045,6 +1055,11 @@
   dependencies:
     lz-string "*"
 
+"@types/node@*":
+  version "20.4.4"
+  resolved "https://registry.yarnpkg.com/@types/node/-/node-20.4.4.tgz#c79c7cc22c9d0e97a7944954c9e663bcbd92b0cb"
+  integrity sha512-CukZhumInROvLq3+b5gLev+vgpsIqC2D0deQr/yS1WnxvmYLlJXZpaQrQiseMY+6xusl79E04UjWoqyr+t1/Ew==
```

Comment:
We might want to lock down the NodeJS version, but that's not necessary in this PR